### PR TITLE
17.1 final branding

### PR DESCRIPTION
**Pull request by Forgind**
...to be merged after 17.1 is updated (+ a little time in case something really high-pri comes in in the next couple weeks)

### CODE REVIEW

**Review by rainersigwald — CHANGES_REQUESTED** [review]
Looks good but per usual I'm requesting changes to reduce the likelihood of an accidental merge.

File: eng/Versions.props [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>17.1.0</VersionPrefix>
+    <VersionPrefix>17.1.0</VersionPrefix><DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```
Comment (Nirmal4G) [inline eng/Versions.props:5]: I've been meaning to ask, why not add this to a new-line instead?

Like so…

```suggestion
    <VersionPrefix>17.1.0</VersionPrefix>
    <DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```

Comment (rainersigwald) [reply eng/Versions.props:5]: When it's on the same line, it's guaranteed to cause a merge conflict with the bump to 17.2. That ensures that when this branch is merged back to `main` we don't accidentally produce a "final release 17.2" build.

Comment (Nirmal4G) [reply eng/Versions.props:5]: 1. Review Block (_on purpose_)
2. Merge Conflict (_on purpose_)

So, we have twice the protection!? 🤣👌

Comment (Forgind) [reply eng/Versions.props:5]: Well, they're protection at different stages. The review block prevents us from merging this PR prematurely, that is, before we feel reasonably confident we're finished with 17.1 (pre-servicing); putting them on the same line ensures that when this merges into main, we don't keep the final branding.

**Review by rainersigwald — APPROVED** [review]

**Review by BenVillalobos — APPROVED** [review]

**Pull request merged**